locales/en/plugin__kubevirt-plugin.json

```diff
@@ -876,6 +876,7 @@
   "Terms": "Terms",
   "The changes you have made require this VirtualMachine to be restarted.": "The changes you have made require this VirtualMachine to be restarted.",
   "The channel to track and receive the updates from.": "The channel to track and receive the updates from.",
```

Comment:
Maybe "The channel used to track and receive updates." I'm not sure what the meaning is supposed to be.